obspy/earthworm/client.py

```diff
@@ -77,22 +77,22 @@ def getWaveform(self, network, station, location, channel, starttime,
         .. rubric:: Example
 
         >>> from obspy.earthworm import Client
-        >>> client = Client("pele.ess.washington.edu", 16017)
-        >>> dt = UTCDateTime(2013, 1, 17) - 2000  # now - 2000 seconds
-        >>> st = client.getWaveform('UW', 'TUCA', '', 'BHZ', dt, dt + 10)
+        >>> client = Client("pubavo1.wr.usgs.gov", 16022)
+        >>> dt = UTCDateTime() - 2000  # now - 2000 seconds
+        >>> st = client.getWaveform('AV', 'ACH', '--', 'EHE', dt, dt + 10)
         >>> st.plot()  # doctest: +SKIP
-        >>> st = client.getWaveform('UW', 'TUCA', '', 'BH*', dt, dt + 10)
+        >>> st = client.getWaveform('AV', 'ACH', '--', 'EH*', dt, dt + 10)
         >>> st.plot()  # doctest: +SKIP
 
         .. plot::
 
             from obspy.earthworm import Client
             from obspy import UTCDateTime
-            client = Client("pele.ess.washington.edu", 16017, timeout=5)
-            dt = UTCDateTime(2013, 1, 17) - 2000  # now - 2000 seconds
-            st = client.getWaveform('UW', 'TUCA', '', 'BHZ', dt, dt + 10)
+            client = Client("pubavo1.wr.usgs.gov", 16022, timeout=5)
+            dt = UTCDateTime() - 2000  # now - 2000 seconds
+            st = client.getWaveform('AV', 'ACH', '--', 'EHE', dt, dt + 10)
             st.plot()
-            st = client.getWaveform('UW', 'TUCA', '', 'BH*', dt, dt + 10)
+            st = client.getWaveform('AV', 'ACH', '--', 'EH*', dt, dt + 10)
```

Comment:
It looks like `getWaveform` does handle empty strings, but `availability` doesn't (which is why I thought it was necessary.)